packages/presentation/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@spinnaker/presentation",
   "description": "Provides re-usable UI components for deck",
-  "version": "0.0.1",
+  "version": "0.0.1-beta.1",
```

Comment:
Not a big deal, but I'm wondering if we should just use `beta` temporarily and update these to proper versions?

Reply:
yeah, after this merges maybe bump it again? IDK